leaf-ui/rappid-extensions/ConfigInspector.js

```diff
@@ -124,11 +124,14 @@ var Config = Backbone.View.extend({
 
     events: {
         'click .config-elements': 'switchConfig',
-        'click .deleteconfig-button': 'removeConfig',
         'click .dropdown-button' : 'toggleDropdown',
         'dblclick .config-elements': 'rename',
         'blur .config-input': 'setConfigName',
-        'keyup .config-input': 'checkForEnter'
+        'keyup .config-input': 'checkForEnter',
+        'click .deleteconfig-button': function(e) {
```

Comment:
Okay this is awesome I didn't realize this was possible. Please share out to the rest of lab because it might be useful in other areas

Reply:
We don't need the e since the event is never used